### PR TITLE
[types/parse] Document Parse.Query.readPreference

### PR DESCRIPTION
**Pull request by mdarens**
See https://parseplatform.org/Parse-SDK-JS/api/3.2.0/Parse.Query.html#readPreference

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/parse/index.d.ts
Consider running `npx dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7db34566eb886906a3352a64d59f4d2a5edc242e
</p>

**Comment by mdarens** [comment]
Ready to merge

File: types/parse/index.d.ts [hunk 1]
```diff
@@ -727,6 +727,15 @@ declare global {
...
         const Installation: InstallationConstructor;
 
+        // Read preference describes how MongoDB driver route read operations to the members of a replica set.
+        enum ReadPreferenceOption {
+            Primary = "PRIMARY",
+            PrimaryPreferred = "PRIMARY_PREFERRED",
+            Secondary = "SECONDARY",
+            SecondaryPreferred = "SECONDARY_PREFERRED",
+            Nearest = "NEAREST",
+        }
```
Comment (MysteryBlokHed) [inline types/parse/index.d.ts:737]: Does this enum actually exist in the code (i.e. is there an object called `ReadPreferenceOption` somewhere)?

File: types/parse/index.d.ts [hunk 2]
```diff
@@ -727,6 +727,15 @@ declare global {
         }
         const Installation: InstallationConstructor;
 
+        // Read preference describes how MongoDB driver route read operations to the members of a replica set.
+        enum ReadPreferenceOption {
```
Comment (sheetalkamat) [inline types/parse/index.d.ts:731]: cannot be enum unless there exists the object with this name in the package to be used.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/parse/index.d.ts (hunk 2 above)
Comment (sheetalkamat) [reply types/parse/index.d.ts:731]: Also documentation you pointed shows function parameters of type string and not special string types so this change looks incorrect

**Pull request closed without merging**